### PR TITLE
Simpler, more direct language added to Scope

### PR DESCRIPTION
**Pull request by cdemwell**

### CODE REVIEW

File: citizen_code_of_conduct.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ If you have been banned from YVRDev and feel that it has been done falsely, emai
 
 We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
 
-This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members. Harassment of our colleagues outside of YVRDerv will not be simply ignored.
```
Comment (garbles) [inline citizen_code_of_conduct.md]: Doesn't this just repeat the previous sentence?

Comment (dlh3) [inline citizen_code_of_conduct.md]: "YVRDe**r**v"

Comment (cdemwell) [inline citizen_code_of_conduct.md]: Fixed typo.

Comment (cdemwell) [inline citizen_code_of_conduct.md]: It's simpler and more direct language - that's the point of this proposal :)

**Pull request merged**